app/views/hub/clients/_navigation.html.erb

```diff
@@ -7,6 +7,7 @@
     <%= tab_navigation_link(t(".client_efile"), efile_hub_client_path(id: @client, anchor: "body")) %>
   <% end %>
   <% if @client.efile_security_informations.size.nonzero? %>
-    <%= tab_navigation_link(t(".client_security"), security_hub_client_path(id: @client)) %>
+    <% fraud_icon = image_tag("security-notification.svg") if @client.efile_security_informations.any?(&:fraud_suspected?) %>
```

Comment:
typically i prefer to keep logic out of views. in part i find it easier to read but it also then means you have the option to write a unit test without using render_views. not blocking!